### PR TITLE
feat(component): add optional text to non required inputs

### PR DESCRIPTION
**Pull request by jorgemoya**
## What
Appends `(optional)` to non required inputs and selects.

## Screenshot
Screen Shot 2019-10-03 at 3 07 24 PM

### CODE REVIEW

File: packages/big-design/src/components/Form/Label/Label.tsx [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
-export type LabelProps = React.LabelHTMLAttributes<HTMLLabelElement>;
+export interface LabelProps extends React.LabelHTMLAttributes<HTMLLabelElement> {
+  renderOptional?: boolean;
+}
 
-export const Label: React.FC<LabelProps> = ({ className, style, ...props }) => <StyledLabel {...props} />;
+export const Label: React.FC<LabelProps> = ({ children, className, renderOptional, style, ...props }) => (
+  <StyledLabel {...props}>
+    {children}
+    {renderOptional && <StyledSpan> (optional)</StyledSpan>}
```
Comment (chanceaclark) [inline packages/big-design/src/components/Form/Label/Label.tsx]: 🍹Why did you end up going with a styled `span` instead of css `content: "(optional)";`?

Comment (jorgemoya) [reply packages/big-design/src/components/Form/Label/Label.tsx]: Didn't even think about it, tbh. I seem to forget about this way of rendering extra text. Seems like a good idea.

Comment (jorgemoya) [reply packages/big-design/src/components/Form/Label/Label.tsx]: Any idea how to test pseudoelements in jest? Can't seem to get it to work.

Comment (chanceaclark) [reply packages/big-design/src/components/Form/Label/Label.tsx]: Beat me to suggesting `toHaveStyleRule`!

**Pull request merged**